erdos/src/dataflow/context.rs

```diff
@@ -144,9 +144,27 @@ where
         &self.config
     }
 
-    /// Get the state attached to the operator.
-    pub fn get_state(&mut self) -> &mut S {
-        &mut self.state
+    /// Get the current state attached to the operator.
+    pub fn get_current_state(&mut self) -> Option<&mut S::Item> {
```

Comment:
I'd like to move away from `get_*` naming, and use `current_state()` instead.

Could you either:
1. Rename the functions, e.g. `get_current_state` -> `current_state`?
2. Stick with the current naming (to keep this PR focused), and make a follow-up PR changing the naming from `get_*()` to `*()`?

Reply:
I'll push out a new PR to do the renaming of the `get` methods.